packages/hoppscotch-backend/src/user/user.service.spec.ts

```diff
@@ -556,4 +641,12 @@ describe('UserService', () => {
       expect(result).toEqual(10);
     });
   });
+
+  describe('removeUsersAsAdmin', () => {
```

Comment:
Please add more test cases for different situations like if userUids provided was not present in the db etc.